models/nft/ethereum/nft_ethereum_mints.sql

```diff
@@ -0,0 +1,75 @@
+with transactions as (
+  select *
+  from {{ source('ethereum', 'transactions')}}
+),
+
+nft_transfers as (
+  select 'erc1155' as erc_standard, *
```

Comment:
can be replaced with nft_ethereum_transfers?

Reply:
make sense. I will use that.